src/middleware/node/parse-request.ts

```diff
@@ -41,7 +44,7 @@ export async function parseRequest(
     let bodyChunks: Uint8Array[] = [];
     request
       .on("error", reject)
-      .on("data", (chunk) => bodyChunks.push(chunk))
+      .on("data", (chunk: any) => bodyChunks.push(chunk))
```

Comment:
Should this be 
```suggestion
      .on("data", (chunk: Uint8Array) => bodyChunks.push(chunk))
```

instead?